db/migrate/20150604110221_acts_as_follower_migration.rb

```diff
@@ -0,0 +1,17 @@
+class ActsAsFollowerMigration < ActiveRecord::Migration
+  def self.up
+    create_table :follows, :force => true do |t|
+      t.references :followable, :polymorphic => true, :null => false
+      t.references :follower,   :polymorphic => true, :null => false
+      t.boolean :blocked, :default => false, :null => false
+      t.timestamps
+    end
+
+    add_index :follows, ["follower_id", "follower_type"],     :name => "fk_follows"
+    add_index :follows, ["followable_id", "followable_type"], :name => "fk_followables"
```

Comment:
Line is too long. [87/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.